services/QuillLMS/app/workers/set_impact_metrics_worker.rb

```diff
@@ -4,30 +4,39 @@ class SetImpactMetricsWorker
   include Sidekiq::Worker
 
   FREE_LUNCH_MINIMUM = 39
+  SENTENCES_PER_ACTIVITY_SESSION = 10
 
   def perform
     set_impact_metrics
   end
 
   def set_impact_metrics
-    finished_activity_sessions_query = QuillBigQuery::ActivitiesAllTimeQuery.run
+    finished_activity_sessions_count = ImpactMetrics::ActivitiesAllTimeQuery.run[0]["count"]
+    active_students_count = ImpactMetrics::ActiveStudentsAllTimeQuery.run[0]["count"]
 
-    teachers_query = QuillBigQuery::ActiveTeachersAllTimeQuery.run
-    teacher_ids = teachers_query.to_a.map {|teacher| teacher["id"]}
+    teachers = ImpactMetrics::ActiveTeachersAllTimeQuery.run
+    teacher_ids = teachers.to_a.map {|teacher| teacher["id"]}
 
-    schools_query = QuillBigQuery::SchoolsContainingCertainTeachersQuery.run(teacher_ids)
-    low_income_schools_query = schools_query.select { |school| school["free_lunches"].present? && school["free_lunches"] > FREE_LUNCH_MINIMUM}
+    schools = ImpactMetrics::SchoolsContainingCertainTeachersQuery.run(teacher_ids)
+    low_income_schools = schools.select { |school| school["free_lunches"].present? && school["free_lunches"] > FREE_LUNCH_MINIMUM}
```

Comment:
Super minor style note.  You could use `school.fetch('free_lunches', 0) > FREE_LUNCH_MINIMUM` instead of `.present? && > FREE_LUNCH_MINIMUM`.  You can take or leave that, I just like to avoid in-line checks for `present?` if possible.